tools/CoqMakefile.in

```diff
@@ -190,6 +190,7 @@ COQMAKEFILE_VERSION:=@COQ_VERSION@
 COQSRCLIBS?= $(foreach d,$(COQ_SRC_SUBDIRS), -I "$(COQLIB)$(d)")
 
 CAMLFLAGS+=$(OCAMLLIBS) $(COQSRCLIBS)
+CAMLFLAGS+=-warn-error +a-3
```

Comment:
What I want is a documented way to disable this.
It could be `make WARN_ERR=` or something else, but there should be a way that does not require me to tamper with CAMLFLAGS that holds other flags. Another way to put it, make gives you `+=` but not `-=` so people cannot easily say "remove that option" unless that option is store in a variable and then they can just set it to empt

Reply:
You can `+= -warn-error -a` which should work fine if it happens after the += though

Reply:
Oh I see what you mean, I think the standard way here is to append more stuff to flags, so you do in your `_CoqProject`:
```
FLAGS+=-warn-error -a
```
and that will take precedence over the previous setting.

Reply:
I am just afraid to add that exact recipe to the readme as then every developer will just do that and we are back to the beginning. But I can add it to the `coq_makefile` doc if you point me where it should be.

Reply:
People are not stupid. If you tell them it is discouraged in published code, they won't do it.

Reply:
Appending a flag that makes another flag ineffective is horrible, depnds on the order things are concatenated, and it depends on the semantics of arguments parsing when multiple instances of the same argument are given.

It is much simpler to have a variable specific to warn-error an blank it out, or replace it by a different value.